client/jetpack-connect/woo-dna-config.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
I think we could add some comments to this file explaining how to extend the usage for new woo flows for later. Here are two comments I'd like to see:

1. One thing we could explain is that the `from` parameter is checked against the params of the object - like "woocommerce-payments". And that we may want to consider changing the structure to include a parameter to check for instead of checking the name in the future, if more complex checks need to be done.

2. Another thing I think should be explained is the difference between a "woo Dna" flow and the other woo flows that exist.